Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
> …issues being discussed are addressed in that patch…

@eboston Yes, your patch is a great improvement.

> regardless of how it works with the menus
> turning clockwise should increase contrast, counter clockwise decrease

The LCD contrast will change according to the same rotation direction as editing values. So clockwise increases contrast, counterclockwise decreases it. Users can reverse the direction of that using `REVERSE_ENCODER_DIRECTION`, if they have some reason to do so.

(Menu navigation direction is reversed with `REVERSE_MENU_DIRECTION`.)